charters/DataBrowser/charter.md

```diff
@@ -0,0 +1,66 @@
+# Data Browser
+
+
+## Description
+The Data Browser (available via the Data Portal 'Explore' section) is an application that provides a faceted search capability allowing users to search and download subsets of cellular resolution data hosted by the Data Coordination Platform (DCP).
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyze, and make available the data in the Human Cell Atlas project.
+
+*Data Portal* The landing site for the Human Cell Atlas (HCA) DCP.
+
+*Analysis Applications* Third party portals listed in the 'Analyze' section of the Data Portal UI.
+
+## Objectives
+The objective of the Data Browser is to provide an easy-to-use User Interface (UI) and RESTful API allowing users to search, download and analyze data stored in the DCP at petabyte scale on public clouds.
+
+## In-scope
+
+### Core capabilities
+* Create indexes to access files in the Data Store by using public Data Store Service (DSS) APIs to read the metadata.
```

Comment:
What is the relationship between the Data Browser and the Query service?

Reply:
The Data Browser receives notifications from the Data Store and constructs its own Indexes. It does not use the Query Service.